keyboards/noxary/268/keymaps/via/rules.mk

```
@@ -0,0 +1,5 @@
+VIA_ENABLE = yes
+MOUSEKEY_ENABLE = no
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+LINK_TIME_OPTIMIZATION_ENABLE = yes
```

Same thing but less unwieldy.

```suggestion
LTO_ENABLE = yes
```

---

And better known.